components/MainPanel.js

```diff
@@ -1,16 +1,37 @@
 import React from 'react';
-import {View, Text, StyleSheet} from 'react-native';
+import { View, Text, StyleSheet } from 'react-native';
 import TodayMain from './TodayMain';
 import TodayPerHour from './TodayPerHour';
 
+const styles = StyleSheet.create({
+  container: {
+    flex: 5,
+    flexDirection: 'column',
+    justifyContent: 'center',
+  },
+  baseText: {
+    fontFamily: 'Roboto',
+    color: '#fff',
+    fontSize: 22,
+  },
+  heading: {
+    textAlign: 'center',
+    fontSize: 35,
+  },
+  thinText: {
+    fontWeight: '100',
+  },
+});
+
 class MainPanel extends React.Component {
   constructor(props) {
     super(props);
     this.state = {};
   }
+
   render() {
-    const weather = this.props.weather;
-    const weatherFetched = this.props.weatherFetched;
+    const { weather } = this.props;
```

Comment:
You can use destructuring creating many variables at once. 
```suggestion
    const { weather, weatherFetched } = this.props;
```